store/tikv/2pc.go

```diff
@@ -833,6 +833,12 @@ type ttlManager struct {
 }
 
 func (tm *ttlManager) run(c *twoPhaseCommitter, lockCtx *kv.LockCtx) {
+	failpoint.Inject("runTTLManager", func() {
+		logutil.BgLogger().Info("[failpoint] injected failure at running ttlManager",
+			zap.Uint64("txnStartTS", c.startTS))
+		failpoint.Return()
+	})
```

Comment:
Is this redundant with `doNotKeepAlive`  in [this pr](https://github.com/pingcap/tidb/pull/22362/files#diff-7352f99d1adcf12eded39ff8934684296725268e9116dcecfa16f560827f9271R842)?

Reply:
Ah, yes. We should do these things in serial.